lib/make-request.js

```diff
@@ -73,6 +73,13 @@ const makeRequest = (instance, { request: requestPhaseList, response: responsePh
       // save reference to the original callback
       const callback = _.get(ctx, 'options.callback', _.noop);
       if (requestPhasesError) {
+        // check if we got a response from caching plugin
```

Comment:
can you also check if it actually is an error object? like `_.isError(requestPhasesError)`